src/devices/Pcd8544/README.md

```diff
@@ -0,0 +1,156 @@
+# PCD8544 - 48 × 84 pixels matrix LCD
+
+This is the famous screen that Nokia 5110 used. It's a SPI based device. This Nokia 5110 was very popular and many of us used to have it in their hands and that young generation have seen it in pictures. This LCD is quite cheap and easy to use.
+
+## Usage
+
+The screen is most of the time presented like this:
+
+![Nokia 5110 screen](./samples/Nokia-5110-LCD-Pinout.png)
+
+It uses SPI and you can add PWM to control the 4 leds brightness which are around the screen. The pin have various names like  BL or LED.
+
+It requires as well a GPIO controller on top of the SPI device with a pin called Data Control, most often presented as DC.
+
+In this schema, the Chip Select (active low) is most of the time called CE. The SPI Clock has to be connected to CLK or SCLK. And the MOSI pin to DIN or MOSI. Names for the pins varies a bit.
+
+Reset pin is sometimes called RST.
+
+Supply voltage can vary, 3.3 or 5V, up to 7V is acceptable. Make sure that there is a current limiting resistor between the brightness led and the controller you'll plug. Most of the kits you buy have it. If you don't add one, you will damage the external brightness leds.
+
+The following example shows how to setup the screen with a PWM to control the brightness:
+
+```csharp
+SpiConnectionSettings spiConnection = new(0, 1) { ClockFrequency = 5_000_000, Mode = SpiMode.Mode0, DataFlow = DataFlow.MsbFirst, ChipSelectLineActiveState = PinValue.Low };
+PwmChannel pwmChannel = PwmChannel.Create(0, 0);
+SpiDevice spi = SpiDevice.Create(spiConnection);
+Pcd8544 lcd = new(27, 22, spi, pwmChannel);
+```
+
+If you don't want neither a PWM neither a reset pin, you can then pass a negative pin number for reset and null for the PWM:
+
+```csharp
+lcd = new(27, -1, spi, null);
+```
+
+## Displaying text
+
+Like for other screen bindings in this repository you have `Write` and `SetCursorPosition` to write text and set the cursor position.
+
+A specific font is available and optimized for the screen. The font only contains a limited number of characters and will just ignore any characters that are unknown.
+
+```csharp
+lcd.SetCursorPosition(0, 0);
+lcd.WriteLine("First line");
+lcd.Write("Second one");
+lcd.SetCursorPosition(0, 5);
+lcd.Write("last line");
+```
+
+The position of the cursor moves with the text you write. The screen has 6 lines in total. And a character is 5 pixels width and 8 pixels height.
+
+You can as well provide your own character using the `Write(ReadOnlySpan<byte> text)` function.
+
+## Drawing lines, rectangles and points
+
+Point, line and rectangle primitives are available:
+
+```csharp
+lcd.DrawPoint(5, 5, true);
+lcd.DrawLine(0, 0, 15, 35, true);
+lcd.DrawRectangle(10, 30, 10, 20, true, true);
+lcd.DrawRectangle(12, 32, 6, 16, false, true);
+// You should not forget to refresh to draw everything
+lcd.Refresh();
```

Comment:
if drawing doesn't happen without calling this, I'd recommend to call it Draw

Reply:
I was debating with myself :-) Thanks, `Draw` it will be!